lms/services/lti_names_roles.py

```diff
@@ -10,6 +10,7 @@
 
 from typing import TypedDict
 
+from lms.models import LTIRegistration
```

Comment:
This code is not really used in the code base yet, from the beginning on the same files (where GH doesn't allow me to post a comment):

```
Service to talk to the Name and Roles LTIA API.
We only implement this now as a way to obtain the LTI Advantage Complete
certification and it's not used anywhere in the codebase yet.	
```